shelley/chain-and-ledger/executable-spec/src/Cardano/Ledger/Val.hs

```diff
@@ -171,3 +136,120 @@ scaledMinDeposit v (Coin mv)
     -- size of the UTxO entry (ie the space the scaled minUTxOValue deposit pays)
     utxoEntrySizeWithoutVal :: Integer
     utxoEntrySizeWithoutVal = inputSize + outputSizeWithoutVal
+
+addrHashLen :: Integer
+addrHashLen = 28
+
+uint :: Integer
+uint = 5
+
+assetIdLen :: Integer
+assetIdLen = 30 -- I have no idea what this is supposed to be
```

Comment:
let's mark these loose ends with a TODO (and then add your GitHub handle as in `-- TODO @TimSheard ...` or `-- TODO git issue #`)